mpas_analysis/__main__.py

```diff
@@ -458,6 +458,10 @@ def run_analysis(config, analyses):  # {{{
 
     mainRunName = config.get('runs', 'mainRunName')
 
+    if len(mainRunName) > 55:
+        print('Warning: The main run name is quite long and will be'
+              'truncated in some plots: \n{}\n\n'.format(mainRunName))
+
```

Comment:
@milenaveneziani, does this seem sufficient?  It will show put before the individual analysis tasks start so it will get printed to the screen, not to the log files.

I didn't add a warning like this if the run name in the control config file is too long.  My assumption is that the user would have already got this warning when they ran the control analysis but they could, in theory, change the name of the run.  Usually, that would be in order to make it shorter.  But I could issue a warning about that, too, if you think it will be an issue often enough to be worth it.